hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/signatures/SignatureVerification.java

```diff
@@ -68,4 +71,52 @@ default Bytes evmAlias() {
     default boolean failed() {
         return !passed();
     }
+
+    /**
+     * Convenience method to create a SignatureVerification that failed
+     *
+     * @param key The key for which verification failed
+     */
+    @NonNull
+    static SignatureVerification failedVerification(@NonNull final Key key) {
+        requireNonNull(key, "Key must not be null");
+        return new SignatureVerification() {
```

Comment:
Why do we have two completely separate anonymous inner class interface implementations instead of a record or static inner class for SignatureVerification?

Reply:
Anonymous classes have some interesting costs to be aware of, and honestly having the record be public isn't all that concerning.  Then again, the entire implementation is here in the interface, so it would be entirely reasonable to make it a class and not even mess with an interface here (as there is unlikely to ever be another implementation).
It's your design; just worth noting that we may be overcomplicating things, and paying a performance cost for that here.